docs/architecture/porting-existing-aspnet-apps/incremental-migration-strategies.md

```diff
@@ -27,7 +27,7 @@ One piece of the .NET ecosystem that helps with interoperability between differe
 
 One approach to incrementally upgrade a .NET Framework 4.5 system layer-by-layer is to first update its class library dependencies to .NET Framework 4.8. Then, modify these libraries to be .NET Standard class libraries. Use multi-targeting and conditional compilation, if necessary. This step can be helpful in scenarios where app dependencies require .NET Framework and cannot easily be ported directly to use .NET Standard and .NET Core. Since .NET Framework libraries can be consumed by ASP.NET Core 2.1 apps, the next step is to migrate some or all of the web functionality of the app to ASP.NET Core 2.1 (as described in the [previous chapter](choose-net-core-version.md)). This is a "bottom up" approach, starting with low level class library dependencies and working up to the web app entry point.
 
-Once the app is running on ASP.NET Core 2.1, migrating it to ASP.NET Core 3.1 in isolation is relatively straightforward. The most likely challenge during this step is updating incompatible dependencies to support .NET Core and possibly higher versions of .NET Standard. For apps that don't have problematic dependencies on .NET Framework-only libraries, there's little reason to upgrade to ASP.NET Core 2.1 or even 3.1. Porting directly to ASP.NET Core 6 makes more sense and requires less effort.
+Once the app is running on ASP.NET Core 2.1, migrating it to .NET 6 in isolation is relatively straightforward. The most likely challenge during this step is updating incompatible dependencies to support .NET Core and possibly higher versions of .NET Standard. For apps that don't have problematic dependencies on .NET Framework-only libraries, there's little reason to upgrade to ASP.NET Core 2.1 or even .NET 6. Porting directly to ASP.NET Core 6 makes more sense and requires less effort.
 
 By the time the app is running on .NET Core 3.1, migrating to the latest .NET release is relatively painless. The process primarily involves updating the target framework of your project files and their associated NuGet package dependencies. While there are several [breaking changes to consider when moving to .NET 6](../../core/compatibility/6.0.md), most apps don't require significant modifications to move from .NET Core 3.1 to .NET 6.
```

Comment:
This paragraph still contains a lot of references to .NET Core 3.1, this should be revised too, right?

Reply:
Yeah right, I believe this paragraph can be removed since it will be no more relevant.

Reply:
@erjain  I agree, yes.